lib/draper/decoratable.rb

```diff
@@ -11,6 +12,10 @@ module Decoratable
     extend ActiveSupport::Concern
     include Draper::Decoratable::Equality
 
+    included do
+      prepend Draper::Compatibility::Broadcastable if defined? Turbo
```

Comment:
I think `Turbo` is too generic.  For example,  [turbo-mount](https://github.com/skryukov/turbo-mount) gem uses the same namespace and it is possible that an application uses it. Can we use a more unique name of turbo-rails?